### ISSUE TITLE
multi account feature 😅

### ISSUE THREAD

**Issue by rinrinx2** [itszechs/DriveStream#38]
It would be great if there was a multi-account feature.  For example, every account we enter appears in a sub-segment so we can add 5 of 10 accounts.

Sorts accounts like xplorer manager

**Comment by Learner00100** [comment]
@itszechs kindly give update on this feature

**Comment by rinrinx** [comment]
bro can you add this feature it's been almost 1 year😁😁😁

**Comment by Learner00100** [comment]
> bro can you add this feature it's been almost 1 year😁😁😁

Yes waiting for developer update, recently many new updates coming he will consider this also

**Comment by rinrinx** [comment]
This is one of the best features the app will be awesome if added
